src/services/data/documents.ts

```diff
@@ -10,14 +10,16 @@ export default (
   httpAuth: HttpInstance
 ): DataDocumentsService => {
   function partialApplyFind(schemaId) {
```

Comment:
Comment on the existing code not on you:

Following the way that we 'should implement' the get endpoints for a service this is WAY off.  I wonder if we should make a ticket to fix this up in the future. Looking at the code for the `addPagersFn`, `findAllGeneric`, etc.. methods, it's actually hard to tell if this is working as intended or by 'accident'.

```ts
  async function query(schemaId, options) {
    const customResponseKeys = ['data.data'];
    const baseConfig = options?.headers ? {headers: options.headers} : {};

    const { data } = await client.get(httpAuth, `/${schemaId}/documents${options?.rql || ''}`, { ...baseConfig, customResponseKeys });
    return data;
  }
```

Reply:
This can be an improvement ticket for the SDK. Do you want to make it?

Reply:
I'll make a ticket 🤗

Reply:
https://extrahorizon.atlassian.net/browse/SVCS-2618